cli/cli/dynamic_configuration/core.py

```diff
@@ -0,0 +1,179 @@
+import importlib
+
+import click
+
+
+def find_question_by_id(questions, question_id):
```

Comment:
Can possibly add some docstring to these  functions. However, the function names are quite clear on themselves as well. Just noticed that in the `predefined_funcs.py` file there are indeed some docstrings.

Reply:
ohh yes, sure. Will do it.

Reply:
done. @Younday